apps/site/assets/ts/schedule/components/schedule-finder/UpcomingDepartures.tsx

```diff
@@ -274,7 +294,8 @@ export const upcomingDeparturesTable = (
                 Destinations
               </th>
               <th scope="col" colSpan={2} className="schedule-table__cell">
-                {isABusRoute(journeysWithTripInfo[0].route) ? (
+                {isABusRoute(journeysWithTripInfo[0].route) &&
```

Comment:
It looks like the `isABusRoute` check is combined with the `someCrowdingInfoExists`  everywhere it's used. Would it makes sense to combine it, so `isABusRoute` is just part of what it means for `someCrowdingInfoExists`?

Reply:
It does make sense to combine that, I'll change it.